2017-09-03 09:37:11 WARNING (MainThread) [homeassistant.setup] Setup of notify is taking over 10 seconds.
2017-09-03 09:37:12 WARNING (MainThread) [homeassistant.setup] Setup of binary_sensor is taking over 10 seconds.
2017-09-03 09:37:12 WARNING (MainThread) [homeassistant.setup] Setup of media_player is taking over 10 seconds.
2017-09-03 09:37:14 WARNING (MainThread) [homeassistant.setup] Setup of device_tracker is taking over 10 seconds.
2017-09-03 09:37:30 WARNING (MainThread) [homeassistant.components.sensor.template] Could not render template Thermostat Battery, the state is unknown.
2017-09-03 09:37:32 WARNING (MainThread) [homeassistant.helpers.condition] Value cannot be processed as a number: 
2017-09-03 09:37:34 WARNING (Thread-4) [requests.packages.urllib3.connectionpool] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', BadStatusLine("''",))': /cgi-bin/snapshot.cgi?channel=1
2017-09-03 09:37:34 WARNING (Thread-4) [requests.packages.urllib3.connectionpool] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', BadStatusLine("''",))': /cgi-bin/snapshot.cgi?channel=1
2017-09-03 09:37:34 WARNING (Thread-4) [requests.packages.urllib3.connectionpool] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', BadStatusLine("''",))': /cgi-bin/snapshot.cgi?channel=1
2017-09-03 09:37:37 WARNING (Thread-2) [requests.packages.urllib3.connectionpool] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPConnectionPool(host='10.40.0.11', port=80): Read timed out. (read timeout=3.0)",)': /cgi-bin/snapshot.cgi?channel=0
2017-09-03 09:38:01 WARNING (Thread-8) [homeassistant.components.zwave] zwave not ready after 30 seconds, continuing anyway
2017-09-03 09:38:18 WARNING (Thread-3) [requests.packages.urllib3.connectionpool] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', BadStatusLine("''",))': /cgi-bin/snapshot.cgi?channel=1
2017-09-03 09:38:18 WARNING (Thread-3) [requests.packages.urllib3.connectionpool] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', BadStatusLine("''",))': /cgi-bin/snapshot.cgi?channel=1
2017-09-03 09:38:18 WARNING (Thread-3) [requests.packages.urllib3.connectionpool] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', BadStatusLine("''",))': /cgi-bin/snapshot.cgi?channel=1
2017-09-03 09:38:18 WARNING (Thread-3) [requests.packages.urllib3.connectionpool] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', BadStatusLine("''",))': /cgi-bin/snapshot.cgi?channel=1
